api_tests/users/views/test_user_detail.py

```diff
@@ -842,8 +842,474 @@ def test_partial_put_user_logged_in(self, app, user_one, url_user_one):
+                                   'ongoing': False,
+                                   'department': '',
+                                   'institution': 'Fake U'
+                                   }]
+                },
+            }
+        }, expect_errors=True)
+        assert res.status_code == 401
+        assert res.json['errors'][0]['detail'] == 'Authentication credentials were not provided.'
+
+    def test_user_put_schools_403(self, app, user_two, url_user_one):
+        res = app.put_json_api(url_user_one, {
+            'data': {
+                'id': user_two._id,
+                'type': 'users',
+                'attributes': {
+                    'full_name': user_two.fullname,
+                    'education': [{'degree': '',
+                                   'startYear': 1991,
+                                   'startMonth': 9,
+                                   'endYear': 1992,
+                                   'endMonth': 9,
+                                   'ongoing': False,
+                                   'department': '',
+                                   'institution': 'Fake U'
+                                   }]
+                },
+            }
+        }, auth=user_two.auth, expect_errors=True)
+        assert res.status_code == 403
+        assert res.json['errors'][0]['detail'] == 'You do not have permission to perform this action.'
+
+    def test_user_put_schools_validation(self, app, user_one, url_user_one):
+        # Tests to make sure schools fields correct structure
+        res = app.put_json_api(url_user_one, {
+            'data': {
+                'id': user_one._id,
+                'type': 'users',
+                'attributes': {
+                    'full_name': user_one.fullname,
+                    'education': {}
+                },
+            }
+        }, auth=user_one.auth, expect_errors=True)
+
+        assert res.status_code == 400
+        assert res.json['errors'][0]['detail'] == 'Expected a list of items but got type "dict".'
+
+        # Tests to make sure structure is lists of dicts consisting of proper fields
+        res = app.put_json_api(url_user_one, {
+            'data': {
+                'id': user_one._id,
+                'type': 'users',
+                'attributes': {
+                    'full_name': user_one.fullname,
+                    'education': [{}]
+                },
+            }
+        }, auth=user_one.auth, expect_errors=True)
+        print res.json
+        assert res.status_code == 400
+        assert res.json['errors'][0]['detail'] == "u'startYear' is a required property"
+
+        # Tests to make sure institution is not empty string
+        res = app.put_json_api(url_user_one, {
+            'data': {
+                'id': user_one._id,
+                'type': 'users',
+                'attributes': {
+                    'full_name': user_one.fullname,
+                    'education': [{'degree': '',
```

Comment:
Again, instead of redefining the whole dict, you could just reassign the institutions key for this test, i.e. 
```
USER_VALID_EDUCATION['institution'] = ''
```